common/workunit/workflow.cpp

```diff
@@ -1079,11 +1072,8 @@ void WorkflowMachine::processSuccessors(CCloneWorkflowItem & item, Owned<IWorkfl
                 //only process the exception if cur is active
                 if(cur.isActive())
                 {
-                    cur.setAbortFlag(item.queryAbortFlag());
                     addToItemQueue(thisWfid);
                     netBranches++;
```

Comment:
Is there a potential problem if the workflow item finishes executing before netBranches is incremented?